packages/js/product-editor/src/blocks/generic/text-area/editor.scss

```diff
@@ -0,0 +1,32 @@
+.wp-block-woocommerce-product-text-area-field {	
+	.rich-text {
+		width: 100%;
+		min-height: calc($gap-larger * 3);
```

Comment:
Seems odd to base the `min-height` on the gap instead of something related to the font size.

Reply:
How do you think we should define the minimum height? Maybe we can use a different var?